lib/FundDistribution/FundDistributionView/FundDistributionView.js

```diff
@@ -14,18 +14,12 @@ import {
 import { AmountWithCurrencyField } from '../../AmountWithCurrencyField';
 import { FUND_DISTR_TYPE } from '../../constants';
 import { fundDistributionShape } from '../../shapes';
+import {
+  FUND_DISTRIBUTION_COLUMN_MAPPING,
+  FUND_DISTRIBUTION_VISIBLE_COLUMNS,
+} from './constants';
 
-const VISIBLE_COLUMNS = ['name', 'expenseClass', 'value', 'amount', 'initialEncumbrance', 'currentEncumbrance'];
```

Comment:
Could you please investigate possible of usage const instead of hard code  (across all pull request)? example below
```
const COLUMNS_NAME = {
  NAME: 'name',
};
const VISIBLE_COLUMNS = [COLUMNS_NAME.NAME];
const columnMapping = {
  [COLUMNS_NAME.NAME]: <FormattedMessage id="stripes-acq-components.fundDistribution.adjustmentDescription" />,
};
```

Reply:
Thanks Dmitry! Updated